pkg/chain/bitcoin/config.go

```diff
@@ -1,7 +1,33 @@
 package bitcoin
 
+import (
+	"fmt"
+
+	"github.com/btcsuite/btcd/chaincfg"
+)
+
 // Config stores configuration related to recovering BTC from a closed keep.
 type Config struct {
 	BeneficiaryAddress string
 	MaxFeePerVByte     int32
+	ChainName          string
```

Comment:
Would it be worth adding `Bitcoin` to the property's name? `BitcoinChain` or `BitcoinChainName`?